VMD_scripts/blobulation.tcl

```diff
@@ -0,0 +1,436 @@
+
+
+proc blobulate {MolID lMin H} {
+	#
+	#	The overarching proc, users use this to run program
+	#
+	#
+	#	Arguments:
+	#	MolID (Integer): An integer that assigns what protein the algorithm looks for 
+	#	lMin (Integer): An integers greater than 1 and less then the legnth of the sequence that determines the minimum length of hblobs
+	# 	H (Float): A float that determines the hydropathy threshold, this determines how hydrophobic something needs to be to be counted
+	#	for an h blob
+	#
+	#	Results:
+	#	The results is a user value applied to the protein of choice the differentiates h blobs, p blobs, and s blobs. 
+	
+	source normalized_hydropathyscales.tcl
+	set checked [checker $MolID $lMin $H]
+	if {$checked == -1} {
+		puts "Variables are incorrect ending program"
+		return  
+		}
+	if {$checked == 1} { 
+		puts "here!"
+		set lower [string tolower $MolID]
```

Comment:
what is 'lower'?

Reply:
Lower is the nocase MolID which allows improper capitalization of "top"

Reply:
Changed variable name to nocaseMolID for better clarity